dashboard/app/models/user.rb

```diff
@@ -2107,6 +2107,14 @@ def can_delete_own_account?
     sections_as_student.empty?
   end
 
+  def shared_sections_with?(other_user)
```

Comment:
Consider renaming this to `shared_sections_with` (without the question mark) since it returns a list of sections (and not a boolean).

Reply:
Whoops, good call